### ISSUE TITLE
Instrument Field in MusicXML Imports

### ISSUE THREAD

**Issue by smorell-1** [musescore/MuseScore#17036]
### Your idea

We want MuseScore files originating with a MusicXML import to use the same process to properly display the instrument selected by MuseScore for consistency.  A [recent PR](https://github.com/musescore/MuseScore/pull/17029) pending approval fixes the process for using the instrument field with scores originating in MuseScore and this update seeks to have MusicXML imports work in the same fashion.

### Problem to be solved

[This fix](https://github.com/musescore/MuseScore/pull/17029) updated how we display selected Instruments in in the Staff/Part Properties when adding parts in MuseScore.  When importing a MusicXML file into this same build we are not able to show the instrument that MuseScore is using to drive the range, transposition and playback defaults properly and instead the Audio Synthesizer/VST being used in the previous software is showing up as the instrument as seen here:

_A file originating in Finale using Aria Player_
<img width="747" alt="Screen Shot 2023-03-27 at 1 33 38 PM" src="https://user-images.githubusercontent.com/128835319/228060245-935edded-4055-49a1-bc19-f0c82de6ef1f.png">

_A file originating in Sibelius using Noteperformer_
<img width="747" alt="Screen Shot 2023-03-27 at 1 34 31 PM" src="https://user-images.githubusercontent.com/128835319/228060288-7eeac9d3-0ab1-4b2b-8dcb-b5015b90e6b9.png">

The above screenshots were created using MS4 this version (not the current production version which will create different problematic results):

OS: macOS 12.4, Arch.: x86_64, MuseScore version (64-bit): 4.1.0-230861626, revision: github-musescore-musescore-71fae17

The expected results would show the selected instrument in the instrument field as you would see if you added a new instrument in MuseScore (using the same approach as the pending PR).

### Additional context

Included in this .zip file are 3 different MusicXML files that will trigger this issue.

[Instrument instability - MusicXML Files.zip](https://github.com/musescore/MuseScore/files/11082859/Instrument.instability.-.MusicXML.Files.zip)

**Comment by rettinghaus** [comment]
Is this still an issue. Testing with 4.5.2 seems to be working as expected.

**Comment by smorell-1** [comment]
Yes, I can't replicate this anymore.  Thanks for checking this out @rettinghaus
